core/chains/evm/logpoller/log_poller.go

```diff
@@ -63,6 +63,72 @@ type LogPoller interface {
 	LogsDataWordBetween(ctx context.Context, eventSig common.Hash, address common.Address, wordIndexMin, wordIndexMax int, wordValue common.Hash, confs Confirmations) ([]Log, error)
 }
 
+// GetLogsBatchElem hides away all the interface casting, so the fields can be accessed more easily, and with type safety
+type GetLogsBatchElem rpc.BatchElem
```

Comment:
nit, but `log_poller.go` is getting a very large file, how about extracting that to a separate file?